Cargo.toml

```diff
@@ -48,7 +49,7 @@ serde_json = "1.0.107"
 serde_urlencoded = "0.7.1"
 snap = "1.1.0"
 strum = { version = "0.25.0", features = ["derive"] }
-subxt = { default-features = false, features = ["jsonrpsee", "native"], version = "0.32.1" }
+subxt = { default-features = false, features = ["jsonrpsee", "native", "unstable-light-client"], version = "0.32.1" }
```

Comment:
How "stable" is the light client support currently ?

Reply:
Maybe not the Light Client itself, but the integration of the Light Client with subxt will take some time to stabilize.